### PR TITLE
[Svace] fix svace issues

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Fix svace issues
- init local variable
- check allocation failure case

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: gst/nnstreamer/tensor_common_pipeline.c [hunk 1]
```diff
@@ -597,6 +597,7 @@ gst_tensor_aggregation_add_data (GHashTable * table, const guint32 key)
   else
     hashkey = key;
   aggr = g_new0 (gst_tensor_aggregation_data_s, 1);
+  g_assert (aggr != NULL);
```
Comment (myungjoo) [inline gst/nnstreamer/tensor_common_pipeline.c:600]: Don't fix this. Mark this "false positive". I have reported this issue to SE (SVACE maintainer) and it appears that they didn't fix it yet; g_new series are supposed to kill the process if g_new fails.

**Review by myungjoo — COMMENTED** [review]
LGTM except for checking g_new0 return.

File: gst/nnstreamer/tensor_common_pipeline.c (hunk 1 above)
Comment (myungjoo) [reply gst/nnstreamer/tensor_common_pipeline.c:600]: Besides, try not to ass g_assert in non-test-case codes. Try to make proper error handling code instead.

Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_common_pipeline.c]: updated

**Review by myungjoo — APPROVED** [review]

**Review by anyj0527 — APPROVED** [review]
LGTM✔

**Pull request merged**